src/app/Core/ExportServices/DataExportServices/generateDataString.service.ts

```diff
@@ -9,18 +9,20 @@ import { generateDataCsvString } from "./generateDataCsvString.service";
  * @desc Returns the table header and the filtered dataset as string to save it as csv
  * @param {string[]} keys - keys of the dataset
  * @param {Record<string, string>[]} data - dataset
+ * @param {MainFilterLabelInterface} mainFilterLabels - Obj with translated labels for the table header

```

Comment:
and DbKey[], DbCollection for key and data

Reply:
Done.